src/components/seller/add-edit-seller/SellerRegistration.tsx

```diff
@@ -53,26 +60,22 @@ const SellerRegistrationForm = () => {
           onChange={handleChange}
         ></textarea>
       </div>
-      <button className={`${BUTTON_CLASS} bg-green-600 text-white mb-4`}>Set Search Centre</button>
+      <div className="flex justify-end">
+      <Button label='Set Search Centre' onClick={() => router.push('/map-center')} />
```

Comment:
Thoughts about using the `Link` component to handle navigation rather than router?
```
<Link href="/map-center">
  <Button label='Set Search Centre'/>
</Link>

```